src/main.rs

```diff
@@ -77,14 +78,20 @@ async fn main() -> Result<(), DeviceManagerError> {
             })?;
     }
 
+    let store = connect_store(&options.store_directory).await?;
```

Comment:
What do you think to move `connect_store` method into `connect` method and remove `store` from the parameters of the `connect` method?

Reply:
This would mean duplicating the code in both the `AstarteDeviceSdk` and `AstarteMessageHub` connect methods. Instead, in this way I create the store only in one place

Reply:
Okay, I don't really agree with you, but leave it at that for now.